datapage.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: data_jp
+title: "Data"
+description: "C-PROOF Glider Data"
+header-img: "img/DataPageHeaderPhoto.jpg"
+---
+<html>
+<style>
+    table{
+        background-color: rgb(250, 250, 250) ;
+        width: 100%;
+        text-align: left;
+
+    }
+</style>
+<body class="datapage"> 
+
+
+<h2>Access Data</h2>
+
+<table >
+  <tr>
+    <td>
+      <p>
+        <a href="{{ site.baseurl }}/data.html"><strong>Direct download</strong></a>: Wget the data for all missions, by glider line, or by platform </br>
+        <a href='{{ site.baseurl }}/deployments/index.html'><strong>Deployments</strong></a>: View and download data and figures for individual glider missions
+      </p>
+    </td>
+  </tr>
+ 
+</table>
+
+
+<h2>Data Description</h2>
+
+The different types of data files that are available are named as follows:
```

Comment:
Can we change this sentence to "As of 2022, the different types of data files that are available are named as follows: " ?
Then I'll put it back after I catch up on updating the old directories to match the new naming conventions.